recipe_batches/recipe_batches.py

```diff
@@ -3,6 +3,9 @@
 import math
 
 def recipe_batches(recipe, ingredients):
+  # compare the keys in ingredients to the keys in recipe
```

Comment:
again, great job. I hope having this plans makes it feel a lot easier to write down the code.